mesheryctl/internal/cli/root/system/start.go

```diff
@@ -449,25 +468,31 @@ func start() error {
 		if err != nil {
 			return err
 		}
-		// Download operator manifest
-		err = utils.DownloadOperatorManifest()
-		if err != nil {
-			return ErrDownloadFile(err, "operator manifest")
-		}
-
-		if !skipUpdateFlag {
-			err = utils.ApplyOperatorManifest(kubeClient, true, false)
 
+		if skipUpdateFlag {
+			err = utils.CanUseCachedOperatorManifests(currCtx)
 			if err != nil {
-				return ErrApplyOperatorManifest(err, true, false)
+				return errors.Wrap(err, "no cached operator manifests")
 			}
-		} else {
+			log.Println("using cached operator manifests...")
 			// skip applying update on operators when the flag is used
 			err = utils.ApplyOperatorManifest(kubeClient, false, false)
```

Comment:
@metonymic-smokey, do we still need to call ApplyOperatorManifest with update = delete = false?
We can skip this call itself, right? Its as good as not applying any newer manifests? Is there anything I am missing here?

Reply:
@hexxdump applying manifests provisions the pods from what I understand. So in order to start Meshery, whether with cached manifests or re-downloaded ones, we need to apply these manifests.

Reply:
We also need to switch to using helm templates, not manifests.

Reply:
@leecalcote yes agreed
for now though, I've stuck to manifests for the time being. Once we switch to helm charts, this could be changed to suit that.

Reply:
Thank you, @metonymic-smokey. Sounds real good.